luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import sys
+
+import luigi
+import hail as hl
+
+from lib.model.mito_mt_schema import SeqrMitoVariantsAndGenotypesSchema, SeqrMitoVariantSchema, SeqrMitoGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+from seqr_loading import SeqrVCFToMTTask
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoVariantMTTask(SeqrVCFToMTTask):
```

Comment:
this needs to inherit from  seqr_loading_optimized. As I have been saying, we only want mito to use the optimized and not the unoptimized

Reply:
Okay. I'm changing it to inherit the `SeqrVCFToVariantMTTask` task.